### ISSUE TITLE
mne_bids_pipeline/_run.py:317: FutureWarning: Setting the `book` attribute is not part of the public API

### ISSUE THREAD

**Issue by hoechenberger** [mne-tools/mne-bids-pipeline#721]
I'm receiving a `FutureWarning`, probably related to Pandas? In any case, we should mask it – it currently appears in the output on the console, which is not nice.
```
mne_bids_pipeline/_run.py:317: FutureWarning: Setting the `book` attribute is not part of the public API, usage can give unexpected or corrupted results and will be removed in a future version
  writer.book = book
```

**Comment by hoechenberger** [comment]
@larsoner Do you know if this has been fixed through #732?

**Comment by larsoner** [comment]
We still have a `writer.book = book` in that file but it's in a `try/except`. We should probably figure out the right way to set the book... or do whatever that code is trying to do.
